src/instructlab/config/init.py

```diff
@@ -70,6 +70,14 @@ def init(
             )
             if not overwrite:
                 return
+        click.secho(
+            """
+            NOTE: ilab no longer supports Python 3.9 or Python 3.12 as of version 0.18.
+            If you are using either of these Python versions, please switch to Python 3.10 or 3.11 to use the latest ilab.
+            You can check which Python version you are using by running `python --version`
+            """,
+            fg="yellow",
+        )
```

Comment:
Suggest adding a line about consequences: "You can continue to run ilab on an unsupported version of Python, but you will be unable to upgrade to more recent versions of ilab with access to bug fixes and new features."

Reply:
Updated the text, lmk what you think!